### ISSUE TITLE
The number

### ISSUE THREAD

**Issue by woolpeeker** [Lyken17/pytorch-OpCounter#28]
https://github.com/Lyken17/pytorch-OpCounter/blob/1f4ddb7fb51c3b1a49d60708f9b857535e5dc4e1/thop/count_hooks.py#L15
I think it is unnecessary to add 1 if a bias exists.
Because it only takes 9 adds to sum 10 numbers up.

https://github.com/Lyken17/pytorch-OpCounter/blob/1f4ddb7fb51c3b1a49d60708f9b857535e5dc4e1/thop/count_hooks.py#L84
I think the batchnorm contains 2*elems MAC operations most.

**Comment by Lyken17** [comment]
> Because it only takes 9 adds to sum 10 numbers up.

That's a good point. But here, it is 9 multiplications + 1 accumulation.

> I think the batchnorm contains 2*elems MAC operations most.

Batch norm 
![image](https://user-images.githubusercontent.com/7783214/61554200-6157c080-aa2a-11e9-9122-c224c6fcbf77.png)
